<launch>

<!-- vehicle name e.g. cc2017_car -->
<arg name="vehicle" default="cc2017_car"/>

<!-- Camera Node -->
<node name="ximea" pkg="ximea_camera" type="ximea_camera_node" output="screen" >
  <rosparam param="camera_param_file_paths" subst_value="true">[$(find ximea_camera)/config/intrinsic_$(arg vehicle).yaml]</rosparam>
  <param name="frame_rate" type="double" value="100.0" />
</node>

<!-- homography publischer -->
<include file="$(find ximea_camera)/launch/homography_publischer_only.launch" />

</launch>
